playbooks/ceph.yml

```diff
@@ -282,8 +284,8 @@
         cifmw_cephadm_dashboard_crt: "{{ cifmw_cephadm_certificate }}"
         cifmw_cephadm_dashboard_key: "{{ cifmw_cephadm_key }}"
 
-
     - name: Create cephfs volume
+      when: cifmw_ceph_daemons_layout.cephfs_enabled  | default(false) | bool
```

Comment:
deploy by default: in case of issues we can change this behavior or override for single node

Reply:
i will make it as defualt(true) for cephfs also